src/dependency-manager/spec/utils/slugs.ts

```diff
@@ -34,6 +34,10 @@ export class Slugs {
   public static ComponentSecretDescription = 'must contain alphanumeric character ([a-z0-9A-Z]), could contain dashes (-), underscores (_), and alphanumerics between.';
   public static ComponentSecretRegexBase = `[a-zA-Z0-9_-]+`;
   public static ComponentSecretValidator = new RegExp(`^${Slugs.ComponentSecretRegexBase}$`);
+
+  public static ComponentSubdomainRegexBase = '([A-Za-z0-9](?:[A-Za-z0-9\\-]{0,61}[A-Za-z0-9])|[\\*|\\@])?';
+  public static ComponentSubdomainDescription = 'must contain alphanumeric character ([a-z0-9A-Z]), could contain dashes (-), underscores (_), and alphanumerics between.';
```

Comment:
🍝 don't just copy and paste. The above regex doesn't support underscores. 

regexr.com/6s3j5

Reply:
Thank you! 'll modify the description to remove the underscore part since it's not part of the regex expression.

Reply:
modified description to remove underscore as a valid character